src/components/authorization/AuthForm.css

```diff
@@ -0,0 +1,87 @@
+/* .main {
+  display: flex;
+  flex-direction: row;
+}
+
+.main section {
+  padding: 2rem;
+  flex-basis: 50%;
+}
+.main img {
+  object-fit: contain;
+} */
+
+.auth-form {
+  width: 100%;
+  max-width: 29rem;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  margin: auto;
+}
+.auth-form label {
+  display: block;
+  margin-bottom: 0.5rem;
+  font-size: 0.8rem;
+}
+.auth-form input {
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.username__wrapper {
+  display: flex;
+  gap: 0.5rem;
+}
+
+/* .form .wrapper-username div {
+  flex-grow: 1;
+} */
+
+/* .form .form__greet-msg {
+  font-size: 1.2rem;
+  color: var(--primary-color-dark);
+  font-weight: bold;
```

Comment:
here too.

Reply:
My bad. I forgot to do that. I'll remove it.